src/main/resources/templates/db_scripts/schema.json

```diff
@@ -929,7 +929,7 @@
     },
     {
       "run":"after",
-      "snippetPath":"setEffectiveLocation.sql",
+      "snippetPath":"setEffectiveHoldingsLocation.sql",
```

Comment:
Why has this file been renamed? Was it to make it less ambiguous which location was being referred to?